Sources/StreamChat/Models/Attachments/AnyAttachmentPayload.swift

```diff
@@ -6,9 +6,22 @@ import Foundation
 
 /// A protocol an attachment payload type has to conform in order it can be
 /// attached to/exposed on the message.
-public protocol AttachmentPayload: Codable {
+public protocol AttachmentPayload: Codable, Equatable {
```

Comment:
Maybe this could be a potential breaking change as well, if for some reason the custom attachment payload of the customer is not automatically equitable (very unlikely as well)